ui/src/views/TcYamlDebug/index.jsx

```diff
@@ -35,16 +35,65 @@ const prefetchSchema = async () => {
     ).json(),
     'github-v1'
   );
+  ajv.addSchema(
+    await (await fetch(urls.schema('queue', 'v1/task-metadata.json'))).json(),
+    'task-metadata.json'
+  );
+  ajv.addSchema(
+    await (await fetch(urls.schema('queue', 'v1/task.json'))).json(),
+    'task.json'
+  );
+  ajv.addSchema(
+    await (
+      await fetch(urls.schema('queue', 'v1/create-task-request.json'))
+    ).json(),
+    'create-task'
+  );
 };
 
 prefetchSchema();
 
+const releaseActions = [
+  'published',
+  'unpublished',
+  'created',
+  'edited',
+  'deleted',
+  'prereleased',
+  'released',
+];
+const pullRequestActions = [
+  'opened',
+  'synchronize',
+  'reopened',
+  'assigned',
+  'auto_merge_disabled',
+  'auto_merge_enabled',
+  'closed',
+  'converted_to_draft',
+  'dequeued',
+  'edited',
+  'enqueued',
+  'labeled',
+  'ready_for_review',
+  'review_requested',
+  'review_request_removed',
+  'unassigned',
+  'unlabeled',
+];
```

Comment:
does this belong in the ui / frontend code? it looks closely tied to github service logic

Reply:
I guess this applies to the whole of UI - it is closely tied to everything already heh :) It also knows about github service and queue service.
Also I didn't see that there are "dictionary endpoints" that return some constants/lists. Do you think we should get one?